NuGet.Config

```diff
@@ -2,7 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="nuget" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
-    <add key="corefx lab" value="https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json" />
+    <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
+    <add key="myget-legacy" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json" />
```

Comment:
What packages restore from this? I would assume we wouldn't need anything from here.

Reply:
myget-legacy feed is for system.commandline. Looks like argumentsyntax class was removed from newer version of system.commandline package. Using myget-legecy to get the version of system.commandline that it was referencing before. We should move to newer version later, then get rid of myget-legacy